README.md

```diff
@@ -141,13 +141,18 @@ where
 
     Value associated with ANY key MAY be NULL.
 
+Example `bmsg` request:
+```bash
+bmsg request fty-info REQUEST INFO 1234
+```
+
 #### HW Capability Request
 
 * HW_CAP/'msg-correlation-id'/'type'
 
 where:
 
-* 'type' can be gpi, gpo, serial (not supported yet)
+* 'type' can be gpi, gpo, type
```

Comment:
, type
can be deleted

Reply:
i suppose that type is obsolete, but the code handle this... so i keep it.